src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallsCollors.values().length);
+        BallsCollors color = BallsCollors.values()[index];
+        return color.toString();
```

Comment:
To get the String representation of an enum, use the name() method instead of toString() to ensure you get the exact name of the enum constant.